### PR TITLE
TOIL is approved by line managers

### PR DESCRIPTION
**Pull request by snim2**

### CODE REVIEW

File: src/staff-handbook/leave/TOIL.md [hunk 1]
```diff
@@ -28,7 +28,8 @@ to take the time back.
 
 ## Requesting TOIL
 
-Using BreatheHR similar to the way that you request holiday, there is a TOIL
-drop down, please write a note as to why you are requesting the leave and this
-will be approved by a member of the People team. Please note this will not deduct
-from your holiday allowance shown on your BreatheHR dashboard.
+Using BreatheHR, go to **My daashboard** and click the link **Log TOIL**,
+fill in the form and click the **Log TOIL** button.
+Please write a note as to why you are requesting the leave and this
```
Comment (rjw1) [inline src/staff-handbook/leave/TOIL.md:33]:
```suggestion
Please write a note as to why you are requesting the TOIL and this
```

File: src/staff-handbook/leave/TOIL.md [hunk 2]
```diff
@@ -28,7 +28,8 @@ to take the time back.
...
+will be approved by your line manager. Once approved, your TOIL will be
+added to your holiday allowance.
```
Comment (rjw1) [inline src/staff-handbook/leave/TOIL.md:35]:
```suggestion
added to your holiday allowance.
You should then book leave as normal to have time not working.
```

File: src/staff-handbook/leave/TOIL.md [hunk 3]
```diff
@@ -28,7 +28,8 @@ to take the time back.
 
 ## Requesting TOIL
 
-Using BreatheHR similar to the way that you request holiday, there is a TOIL
-drop down, please write a note as to why you are requesting the leave and this
-will be approved by a member of the People team. Please note this will not deduct
-from your holiday allowance shown on your BreatheHR dashboard.
+Using BreatheHR, go to **My daashboard** and click the link **Log TOIL**,
+fill in the form and click the **Log TOIL** button.
```
Comment (yndajas) [inline src/staff-handbook/leave/TOIL.md:32]:
```suggestion
fill in the form and click the **log TOIL** button.
```

File: src/staff-handbook/leave/TOIL.md [hunk 4]
```diff
@@ -28,7 +28,8 @@ to take the time back.
 
 ## Requesting TOIL
 
-Using BreatheHR similar to the way that you request holiday, there is a TOIL
-drop down, please write a note as to why you are requesting the leave and this
-will be approved by a member of the People team. Please note this will not deduct
-from your holiday allowance shown on your BreatheHR dashboard.
+Using BreatheHR, go to **My daashboard** and click the link **Log TOIL**,
```
Comment (yndajas) [inline src/staff-handbook/leave/TOIL.md:31]:
```suggestion
Using BreatheHR, go to **My dashboard** and click the link **Log TOIL**,
```

File: src/staff-handbook/leave/TOIL.md (hunk 1 above)
Comment (yndajas) [inline src/staff-handbook/leave/TOIL.md:33]: Can this line either have an extra line break or move up to the previous line?

**Review by yndajas — COMMENTED** [review]
Thanks for doing this!

File: src/staff-handbook/leave/TOIL.md (hunk 3 above)
Comment (yndajas) [reply src/staff-handbook/leave/TOIL.md:32]: (the button has lower case "log")

File: src/staff-handbook/leave/TOIL.md [hunk 5]
```diff
@@ -28,7 +28,9 @@ to take the time back.
...
-Using BreatheHR similar to the way that you request holiday, there is a TOIL
-drop down, please write a note as to why you are requesting the leave and this
-will be approved by a member of the People team. Please note this will not deduct
-from your holiday allowance shown on your BreatheHR dashboard.
+Using BreatheHR, go to **My dashboard** and click the link **Log TOIL**,
+fill in the form and click the **log TOIL** button.
+Please write a note as to why you are requesting the TOIL and this
+will be approved by your line manager.
+Once approved, your TOIL will be added to your holiday allowance.
+You should then book leave as normal to have time not working.
```
Comment (yndajas) [inline src/staff-handbook/leave/TOIL.md:36]: Is there a reason for the line breaks? It condenses into one paragraph without double line breaks, but is it to divide each point up in the code?

**Review by yndajas — APPROVED** [review]
Looks good, just one query left

File: src/staff-handbook/leave/TOIL.md (hunk 5 above)
Comment (snim2) [reply src/staff-handbook/leave/TOIL.md:36]: Not really. In my last place I followed a rule of "one sentence per line" which makes code review much easier, but I'll tidy this up before merging.

**Comment by snim2** [comment]
Rebased.

**Pull request merged**